entries/d/druide.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Druide": {
+    "domain": "druide.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.antidote.info/en/druide-account/client-portal-user-guide/druide-account/safeguarding-your-account#two-factor_authentication",
+    "recovery": "https://www.antidote.info/en/druide-account/client-portal-user-guide/druide-account/safeguarding-your-account#having-issues-with-your-authentication-code",
```

As the page only contains information about 2FA, just linking to the page instead of a specific section is enough.
```suggestion
    "documentation": "https://www.antidote.info/en/druide-account/client-portal-user-guide/druide-account/safeguarding-your-account",
```